images/toolbox/toolbox.sh

```diff
@@ -62,6 +62,9 @@ if [[ -z ${ROOK_API_SERVER_ENDPOINT} ]]; then
         # we have some rook env vars set, get client info from the rook API server and construct
         # a ceph config file from that
         ROOK_API_SERVER_ENDPOINT=${ROOK_API_SERVICE_HOST}:${ROOK_API_SERVICE_PORT}
+    else
+        # hostNetwork: true is most likely used, use DNS name for rook API server access
+        ROOK_API_SERVER_ENDPOINT=rook-api:8124
```

Comment:
It also needs to set:
ROOK_API_SERVICE_HOST=rook-api
ROOK_API_SERVICE_PORT=8124
or rookctl will continue to complain about those variables not being set.